help/markdown/core-commandlineparsing.md

```diff
@@ -40,12 +40,12 @@ A more sophisticated example can be found in the fake runner: https://github.com
 
 Or the target module:
 
-- https://github.com/fsharp/FAKE/blob/rc_1/src/app/Fake.Core.Target/Target.fs#L9-L26
+- https://github.com/fsharp/FAKE/blob/64d871f5065412fe7b233025e454ccf3b89e46d7/src/app/Fake.Core.Target/Target.fs#L9-L26
```

Comment:
These links point to a specific commit. Can we have a permanent link to the latest commit on the master branch maybe?

Reply:
That was on purpose, to avoid unintended 404s or wrong highlighted lines if the code is updated.  

On second thought, I'm not entirely sure what's preferable between a correct link to a potentially not up-to-date sample, or a might-be-broken link to an up-to-date sample...

Do you still think a link to master is better?

Reply:
Ok, so as I saw, the purpose of the links is to provide the user with advanced examples of FAKE command line parsing. Why note we provide extract these examples and inline them in this MD file instead of linking it to. What do u think?

Reply:
Indeed, that would work for the example yes.

And I guess I can point the test suite link to master, as it's less a problem if the lines change.

Reply:
@yazeedobaid Hey, I was about to do what we said, but it doesn't feel right:
- The examples are quite big, as they are meant to be "more sophisticated" than the basic example inlined, and they are actually split in multiple parts. I'm not 100% sure what's important and what's not. I'd rather highlight the main part but keep the rest too as-is, but I think it would be too long to be inlined.
- There is already an inlined example to demonstrate basic usage of the parser, so adding another big one feels redundant.
- I think it's better to have a link to some "real" code on github where we can be reasonably sure that it worked at some point in time, in contrast to an artificial inlined example extracted from the sources that no one compiled before.
- For the same reason, I think it's better to have a permalink to some commit rather than point to master. I checked on master, and this code has moved quite a lot since the permalink, and it's hard to find the originally highlighted part. Since this is only for example purposes, I think it's better to have something that's not completely up to date but is easily understandable.

Sorry for the turnaround.  
I'll at least add a note indicating that the links are probably old and that it would be a good idea to check the current version too.

I'll rebase my branch too, since it's behind by a few commits. I hope it's ok with you.